src/interfaces/HposInterface.ts

```diff
@@ -154,6 +155,16 @@ export interface Transaction {
   expiration_date: string
 }
 
+export interface Redemption {
```

Comment:
We might need to adjust this a bit and include an email.  Let's sync up / or DM and iron this out.